contracts/SmartHoldETH.sol

```diff
@@ -6,7 +6,7 @@ import "./PriceFeedInterface.sol";
 
 contract SmartHoldETH {
   address public owner = msg.sender;
-  uint public depositedAt = block.timestamp;
+  uint public depositedAt = now;
```

Comment:
`now` is deprecated in favour of `block.timestamp` in future versions of solidity

Reply:
Ohhh. Glad to hear that, I didn't know. Will do!